src/store/modules/checkin.js

```diff
@@ -19,16 +19,16 @@ const state = {
 const mutations = make.mutations(state)
 const actions = {
     ...make.actions(state),
-    async dailyCheckin({ state, commit }, payload) {
-        let res = await axios
+    async dailyCheckin({ commit, dispatch }, payload) {
+        let token = localStorage.getItem('authenticated') 
+        await axios
             .post('http://5.9.144.226:8000/checkin', payload, {
                 headers: {
-                    'Authorization': 'Bearer ' + localStorage.getItem('authenticated')
+                    'Authorization': 'Bearer ' + token
```

Comment:
make this global. we will not add this everytime. its should be in global axios setup

api url should be in evn file

use vuex